pvlib/pvsystem.py

```diff
@@ -2274,22 +2303,30 @@ def adrinverter(v_dc, p_dc, inverter, vtol=0.10):
     mppt_hi = inverter['MPPTHi']
     mppt_low = inverter['MPPTLow']
 
-    v_lim_upper = np.nanmax([v_max, vdc_max, mppt_hi])*(1+vtol)
-    v_lim_lower = np.nanmax([v_min, mppt_low])*(1-vtol)
-
-    pdc = p_dc/p_nom
-    vdc = v_dc/v_nom
-    poly = np.array([pdc**0, pdc, pdc**2, vdc-1, pdc*(vdc-1),
-                     pdc**2*(vdc-1), 1/vdc-1, pdc*(1./vdc-1),
-                     pdc**2*(1./vdc-1)])
+    v_lim_upper = np.nanmax([v_max, vdc_max, mppt_hi]) * (1 + vtol)
+    v_lim_lower = np.nanmax([v_min, mppt_low]) * (1 - vtol)
+
+    pdc = p_dc / p_nom
+    vdc = v_dc / v_nom
+    with np.errstate(invalid='ignore', divide='ignore'):
```

Comment:
# zero voltage will lead to division by zero, but since power is set to night time value later, errors can be ignore